### ISSUE TITLE
Multiple unsubscribe request is failing address validation

### ISSUE THREAD

**Issue by nwkeeley** [bojand/mailgun-js#203]
Mailgun-js: 0.17.0
Node.js  8.5 mac osx

This Does not work
```
const mailgun = require('mailgun-js')({ apiKey: MyKey, domain: MyDomain }); 

const data = [
  { address: 'foo@example.com' },
  { address: 'foo2@example.com' }
];

mailgun.post(`/${MyDomain}/unsubscribes`, data, (error, body) => {
  // Fails with error "Error: Missing mandatory parameter: address"
});
```

This does work
```
const mailgun = require('mailgun-js')({ apiKey: MyKey, domain: MyDomain }); 

const data = { address: 'foo@example.com' };

mailgun.post(`/${MyDomain}/unsubscribes`, data, (error, body) => {
  // works
});
```

**Comment by nwkeeley** [comment]
Documentation says to pass a content-type of application/json for multiple requests - but I don't see where / how to pass that header in the current api call

**Comment by bojand** [comment]
Hello, this should be addressed in 0.18.0.

**Comment by nwkeeley** [comment]
Thats great thank you for the quick turn around @bojand
